ImmaculateHangover/crunchy-tunes
# Resolves #32, Resolves #31, Resolves #26

#### Checklist

Mark an X inside of 
- [x] Did I cut my work branch off of master (don't cut new branches from existing feature branches)?
- [X] Did I follow the correct naming convention for my branch?
- [ ] Is my branch focused on a single main change?
- [X] Do all of my changes directly relate to this change?
- [X] Did I rebase the upstream master branch after I finished all my work?
- [X] Did I write a clear pull request message detailing what changes I made?
- [X] Are all tests passing
- [x] Did I write new tests for new functionality?
#### What are the important parts of the code?

In contrast to the general guidelines for our pull requests, this instance is comprised of many changes. Yet, the important parts of this code are all in the view-layer React components. This pull request covers the integration of React Toolbox with our view layer, and the addition of nearly all of the view components. 
#### How should this be tested by the reviewer?

The best way for you test this code is to load our web application and see if the page renders. If there are any issues within any of the view components, then the app will not render.
#### Is any other information necessary to understand the files?

No, but ask me if you have any questions.
#### Attach relevant screenshots (if any)
